components/Button/src/index.css

```diff
@@ -96,6 +96,21 @@
   padding-inline-start: var(--denhaag-button-large-size-padding-inline, var(--denhaag-button-padding-inline));
 }
 
+.denhaag-button--outline-dark-action {
+  border-style: solid;
+  background-color: var(--denhaag-button-outline-dark-action-background-color);
```

Comment:
There is a fallback color missing, when the token does not load, like this:
`background-color: var(--denhaag-button-outline-dark-action-background-color, #123456);`

Reply:
Is there need for a fallback here? If you look at the other styles for the button you'll notice most don't have a fallback, because we simply require the tokens to be filled if an organization wants to use that style.